routes/web.php

```diff
@@ -17,6 +18,9 @@
 
 Route::middleware("auth")->group(function (): void {
     Route::post("/logout", [AuthController::class, "logout"])->name("logout");
+    Route::post("/favorites", [FavoritesController::class, "store"]);
+    Route::get("/favorites/{city_id}", [FavoritesController::class, "check"]);
+    Route::get("/user", fn() => auth()->user());
```

Comment:
For what this route is needed?

Are you sure you want to return the whole instance of the user? It might lead to sensitive data leak to the frontend if somebody forgot to add new property to `hidden` array.
https://laravel.com/docs/10.x/eloquent-serialization#hiding-attributes-from-json

Lets use resource and define what exactly you want to return to the frontend.
https://laravel.com/docs/10.x/eloquent-resources#main-content

P.S.
Better way to determine if user is logged in is `Auth::check()`
https://laravel.com/docs/10.x/authentication#determining-if-the-current-user-is-authenticated

Reply:
Removed